script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -107,16 +107,16 @@ I'm not kidding when I say that your eyes really are that bad!"
 "――――――"
 <Page32>
 "Listen. Everything in this world is predestined for extinction.
-Even the strongest and the longest life must abide by this rule. Even if there is someone that has a lifespan longer than the lifespan of the universe, it doesn't mean that they will keep living forever."
+Even the strongest and the longest life must abide by this rule. Even if there is someone that has a lifespan longer than that of the universe, it doesn't mean that they keep on living forever."
 <Page33>
 "This pre-determined 'annihilation' is the 'death' of all things.
-Since it has always been there, if you have the power to understand the concept of 'death' and posess a Circuit between your brain and eyes, it is not impossible for you to see it.
+Since it has always been there, if you have the power to understand the concept of 'death' and possess a Circuit between your brain and eyes, it is not impossible for you to see it.
 <Page34>
 "That is the true nature of the 'lines' that you see.
 If I were to theorize about it in human terms, I'd say that it's something resembling an intrinsic flaw in atomic bonds.
-Or maybe it's more like a <self-destructive|t e l o m e r e > switch in genes that manifests the cause of an individual's death."
+Or maybe it's more like a telomere, or self-destructive genetic switch, that manifests the cause of an individual's death.
 <Page35>
-"Anyway, what you're seeing are the 'lines where things can easily die', the conclusion that is death itself.
+"Anyway, what you're seeing are the 'lines where things can easily die', the conclusion that is death itself.//to be rechecked
```

Comment:
Anyway, what you're seeing are the 'lines along which things are easily killed', the embodiment of death itself.